src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -235,6 +246,15 @@ private string GetSuffix(Language language, int copy, bool multipleCopies = fals
                 suffixBuilder.Append(IsoLanguages.Get(language).TwoLetterCode);
             }
 
+            if (languageTags.Any())
+            {
+                foreach (var tag in languageTags)
```

Comment:
I think you can shorten to:

```
suffixBuilder.Append(".");
suffixBuilder.Append(string.Join('.', languageTags));
```

Reply:
Indeed, refactored in [3b8ce68](https://github.com/Sonarr/Sonarr/pull/4924/commits/3b8ce68c56e17d8f09dda428c9bdc499e31e99a3)